src/geopathfinder/naming_conventions/yeoda_naming.py

```diff
@@ -289,21 +287,8 @@ def yeoda_path(root, product=None, data_version=None, datalog='datasets', grid=N
         Object for the path
     """
 
-    # get logfile path
-    if datalog == 'datasets':
-        pass
-    elif datalog == 'logfiles':
-        data_version = 'logfiles'
-        grid = None
-        tile = None
-        qlook = False
-    else:
-        raise ValueError('Wrong input for "datalog" level!')
-
     # define the data_version and run number folder name
-    if data_version == 'logfiles':
-        pass
-    elif data_version is not None:
+    if data_version is not None:
```

Comment:
This can be removed since `data_version` is now a positional argument - it should not be `None`.